app/src/main/java/com/bytehamster/flowitgame/GLRenderer.java

```diff
@@ -69,6 +69,8 @@ public void onSurfaceCreated(GL10 gl, EGLConfig config) {
         gl.glGenTextures(textures.length, textures, 0);
         loadTexture(gl, 0, textureDrawables[currentColorschemeIndex]);
 
+        gl.glLoadIdentity();
```

Comment:
Shouldn't the other one be enough? That's the only place where the view matrix is modified.

Reply:
No, you are correct. I had mistakenly thought it was relevant, but it was not. I think my OpenGL skills are at a kind of embarrassing level :)

Reply:
> it does seem at least like less of a bug than the blank screen....

Yeah, definitely. And the fix is so simple and clean, good find!